projects/js-packages/licensing/components/activation-screen-success-info/style.scss

```diff
@@ -8,7 +8,7 @@
 	justify-content: space-between;
 
 	h1 {
-		font-size: var( --font-title-large );
+		font-size: 44px;
```

Comment:
Since there is no variable for 44px, we need to add the `//typography-exception` comment over this line as explained in https://github.com/Automattic/wp-calypso/blob/5b50d26ba437e421bd6942aa91672e78d02d5ad8/packages/typography/styles/variables.scss#L31.